### PR TITLE
Jormungandr: add is_journeys_schedules for stif

### PR DESCRIPTION
**Pull request by xlqian**
https://jira.canaltp.fr/browse/NAVITIAII-2230

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 1]
```diff
@@ -479,6 +481,11 @@ def _set_specific_params(mod):
         else:
             _set_specific_params(default_values)
 
+        # set parameters when is_journey_schedules is set to True
+        if args.get("is_journey_schedules"):
+            args["min_nb_journeys"] = 5
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: I don't think this should override the `min_nb_journeys`

And I'm not sure we should add this parameter under the hood, I would prefer this to be explicit in the `same_journey_schedule`

Comment (xlqian) [reply source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: @TeXitoi

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 2]
```diff
@@ -321,7 +321,8 @@ def wrapper(*args, **kwargs):
 
                     args['min_nb_transfers'] = journey['nb_transfers']
                     args['direct_path'] = 'only' if 'non_pt' in journey['tags'] else 'none'
-                    args['min_nb_journeys'] = 5
+                    args["min_nb_journeys"] = 5
```
Comment (pbougue) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: minor: you can let the simple quotes

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 3]
```diff
@@ -411,6 +412,8 @@ def __init__(self):
         parser_get.add_argument("count", type=default_count_arg_type)
         parser_get.add_argument("_min_journeys_calls", type=int)
         parser_get.add_argument("_final_line_filter", type=boolean)
+        parser_get.add_argument("is_journey_schedules", type=boolean, default=False,
+                                description="True when '/journeys' is called to compute the same journey schedules")
```
Comment (pbougue) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: maybe we can add that it (de)activates appropriate parameters for that use.

**Review by pbougue — APPROVED** [review]

**Review by antoine-de — APPROVED** [review]

**Pull request merged**